diff -u -r LimeSurvey-3.17.7-190627/application/views/admin/authentication/ajaxLogin.php LimeSurvey-3.17.7-190627.updated/application/views/admin/authentication/ajaxLogin.php
--- LimeSurvey-3.17.7-190627/application/views/admin/authentication/ajaxLogin.php	2019-06-27 15:37:37.000000000 +0200
+++ LimeSurvey-3.17.7-190627.updated/application/views/admin/authentication/ajaxLogin.php	2020-04-03 00:44:52.269238522 +0200
@@ -54,8 +54,11 @@
                                     'options' => array(
                                             'onChange'=>'this.form.submit();'
                                             )
-                            )));
-
+                                ),
+                                'events' => array(
+                                    'select2:select' => 'function (e) {this.form.submit();}'
+                                )
+                            ));
 
                         }
                         else
diff -u -r LimeSurvey-3.17.7-190627/application/views/admin/authentication/login.php LimeSurvey-3.17.7-190627.updated/application/views/admin/authentication/login.php
--- LimeSurvey-3.17.7-190627/application/views/admin/authentication/login.php	2019-06-27 15:37:37.000000000 +0200
+++ LimeSurvey-3.17.7-190627.updated/application/views/admin/authentication/login.php	2020-04-03 00:45:38.910315243 +0200
@@ -39,7 +39,6 @@
                                     // Make sure we have a default auth, if not set, use the first one we find
                                     $defaultAuth = reset($pluginNames);
                                 }
-
                                 if (count($pluginContent)>1)
                                 {
                                     $selectedAuth = App()->getRequest()->getParam('authMethod', $defaultAuth);
@@ -67,7 +66,11 @@
                                             'options' => array(
                                                     'onChange'=>'this.form.submit();'
                                                     )
-                                    )));
+                                        ),
+                                        'events' => array(
+                                            'select2:select' => 'function (e) {this.form.submit();}'
+                                        )
+                                    ));
 
 
                                 }
