diff -u -r LimeSurvey-4.2.3-200511/application/helpers/remotecontrol/remotecontrol_handle.php LimeSurvey-4.2.3-200511.update/application/helpers/remotecontrol/remotecontrol_handle.php
--- LimeSurvey-4.2.3-200511/application/helpers/remotecontrol/remotecontrol_handle.php  2020-05-11 02:08:48.000000000 +0200
+++ LimeSurvey-4.2.3-200511.update/application/helpers/remotecontrol/remotecontrol_handle.php   2020-06-23 02:02:58.137100900 +0200
@@ -255,13 +255,23 @@
     /**
      * Copy survey (RPC function)
      *
+     * Available options are:
+     * * copysurveyexcludequotas
+     * * copysurveyexcludepermissions
+     * * copysurveyexcludeanswers
+     * * copysurveyresetconditions
+     * * copysurveyresetstartenddate
+     * * copysurveyresetresponsestartid
+     *
      * @access public
      * @param string $sSessionKey Auth credentials
      * @param int $iSurveyID_org Id of the source survey
      * @param string $sNewname name of the new survey
+     * @param array|null $aOptions (optional) List of options to customize the cloning process.
+     *  *
      * @return array On success: new $iSurveyID in array['newsid']. On failure array with error information
      * */
-    public function copy_survey($sSessionKey, $iSurveyID_org, $sNewname)
+    public function copy_survey($sSessionKey, $iSurveyID_org, $sNewname, $aOptions = null)
     {
         $iSurveyID = (int) $iSurveyID_org;
         if (!$this->_checkSessionKey($sSessionKey)) {
@@ -280,7 +290,36 @@
         } else {
             $aExcludes = array();
             $sNewSurveyName = $sNewname;
-            $aExcludes['dates'] = true;
+
+            // Exclude list initialization based on options
+            if(isset($aOptions) && is_array($aOptions)){
+                if (in_array('copysurveyexcludequotas', $aOptions)) {
+                    $aExcludes['quotas'] = true;
+                }
+
+                if (in_array('copysurveyexcludepermissions', $aOptions)) {
+                    $aExcludes['permissions'] = true;
+                }
+
+                if (in_array('copysurveyexcludeanswers', $aOptions)) {
+                    $aExcludes['answers'] = true;
+                }
+
+                if (in_array('copysurveyresetconditions', $aOptions)) {
+                    $aExcludes['conditions'] = true;
+                }
+
+                if (in_array('copysurveyresetstartenddate', $aOptions)) {
+                    $aExcludes['dates'] = true;
+                }
+
+                if (in_array('copysurveyresetresponsestartid', $aOptions)) {
+                    $aExcludes['reset_response_id'] = true;
+                }
+            }else{
+                $aExcludes['dates'] = true;
+            }
+
             $btranslinksfields = true;
             Yii::app()->loadHelper('export');
             $copysurveydata = surveyGetXMLData($iSurveyID, $aExcludes);
